src/deluge/gui/ui/menus.cpp

```diff
@@ -1100,21 +1101,13 @@ defaults::Velocity defaultVelocityMenu{STRING_FOR_VELOCITY, STRING_FOR_DEFAULT_V
 defaults::Magnitude defaultMagnitudeMenu{STRING_FOR_RESOLUTION, STRING_FOR_DEFAULT_RESOL_MENU_TITLE};
 defaults::BendRange defaultBendRangeMenu{STRING_FOR_BEND_RANGE, STRING_FOR_DEFAULT_BEND_R};
 defaults::MetronomeVolume defaultMetronomeVolumeMenu{STRING_FOR_METRONOME, STRING_FOR_DEFAULT_METRO_MENU_TITLE};
-
+defaults::PadBrightness defaultPadBrightness{STRING_FOR_DEFAULT_PAD_BRIGHTNESS,
+                                             STRING_FOR_DEFAULT_PAD_BRIGHTNESS_MENU_TITLE};
 Submenu defaultsSubmenu{
     STRING_FOR_DEFAULTS,
-    {
-        &defaultUI,
-        &defaultAutomationMenu,
-        &defaultTempoMenu,
-        &defaultSwingMenu,
-        &defaultKeyMenu,
-        &defaultScaleMenu,
-        &defaultVelocityMenu,
-        &defaultMagnitudeMenu,
-        &defaultBendRangeMenu,
-        &defaultMetronomeVolumeMenu,
-    },
+    {&defaultUI, &defaultAutomationMenu, &defaultTempoMenu, &defaultSwingMenu, &defaultKeyMenu, &defaultScaleMenu,
+     &defaultVelocityMenu, &defaultMagnitudeMenu, &defaultBendRangeMenu, &defaultMetronomeVolumeMenu,
+     &defaultPadBrightness},
```

Comment:
trailing comma after `&defaultPadBrightness` needed here so it doesn't reformat the whole block

Reply:
fixed :)